zombie/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+  
+$engine: 2
+$minFxVersion: 3362
+name: Zombie
+version: v1.0.0
+author: Hamy-os
+description: This is a zombie rpg framework built for fivem
+    
+tasks:
+  # Download default CFX resources
+  - action: download_github
+    src: https://github.com/citizenfx/cfx-server-data
+    ref: 88f4f974b855e0bf2ca5f51b9916648bd5b7d44b
+    subpath: resources
+    dest: ./resources
+  
+    # Download zombie
+  - action: download_github
+    src: https://github.com/Dislaik/zombieoutbreak
+    dest: ./resources/zombie
+
+  # Download and prepare server.cfg  / database
+  - action: download_github
+    src: https://github.com/tabarra/txAdmin-recipes/

```

Comment:
use subpath

Reply:
there is no reason to i like it this way